tests/test_ilastik/test_experimental/types.py

```diff
@@ -29,7 +31,10 @@ class TestProjects(enum.Enum):
 @dataclasses.dataclass
 class Dataset:
     path: str
+    #: Axes to use for api prediction
     axes: str
+    #: Axes to use for headless run
+    headless_axes: str
```

Comment:
When running headless project (trained on xy image) on the image (x, y, c) where c is 1 it will fail. So for such case axis order is explicitly specified

Reply:
headless refers to the "headless" mode in ilastik, and is not a good fit here. Also I don't find it very intuitive in this context. Maybe something like `training_axes` or `project_axes` would be a better fit.

Reply:
Those axis not about project, but about dataset. This what would be passed to `--input_axes` during the headless prediction run.

Reply:
as far as I understand the `--input_axes` parameter it should represent the axes in the data. Are you adding singleton `c` here?

Reply:
Not exactly as it's a png it's already there. Axes of dataset when added to ilastik are (x, y, c = 1) but if I try to run project trained on (x, y) axes without explicitly specifying input axes (x, y, c) using this dataset it will fail with an error.

Reply:
ah, alright, then it's fine. I'd still not call it `headless_axes` but `data_axes` then